linera-sdk/Cargo.toml

```diff
@@ -48,6 +48,10 @@ serde.workspace = true
 serde_json.workspace = true
 wit-bindgen.workspace = true
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+anyhow.workspace = true
+tokio = { workspace = true, features = ["rt", "macros"] }
```

Comment:
These dependencies should not be included when building for Wasm. You can instead mark the test to only run on non-Wasm targets, using `#[cfg(not(target_arch = "wasm"))]`.